### ISSUE TITLE
synergyc fails to start

### ISSUE THREAD

**Issue by koppi** [symless/synergy#5322]
```bash
$ synergyc --version
synergyc 1.4.16, protocol version 1.5
Copyright (C) 2012 Bolton Software Ltd.
Copyright (C) 2008-2012 Nick Bolton
Copyright (C) 2002-2012 Chris Schoeneman
```

```bash
$ DISPLAY=:0 synergyc --no-daemon -d DEBUG2 192.168.1.1
2016-05-11T23:08:06 INFO: Synergy 1.4.16 Client on Linux 4.1.12-rt13-CX9020-9+ #1 PREEMPT RT Fri Apr 29 01:09:47 CEST 2016 armv7l
	/build/synergy-M6oALT/synergy-1.4.16/src/lib/synergy/CClientApp.cpp,157
2016-05-11T23:08:06 DEBUG1: starting client
	/build/synergy-M6oALT/synergy-1.4.16/src/lib/synergy/CClientApp.cpp,633
2016-05-11T23:08:06 DEBUG: XOpenDisplay(":0")
	/build/synergy-M6oALT/synergy-1.4.16/src/lib/platform/CXWindowsScreen.cpp,918
2016-05-11T23:08:06 DEBUG1: thread 0x00000002 entry
	/build/synergy-M6oALT/synergy-1.4.16/src/lib/mt/CThread.cpp,155
X Error of failed request:  BadValue (integer parameter out of range for operation)
  Major opcode of failed request:  53 (X_CreatePixmap)
  Value in failed request:  0x0
  Serial number of failed request:  16
  Current serial number in output stream:  17
```

```bash
$ synergyc --no-daemon -d DEBUG2 --display :0 192.168.1.1
2016-05-11T23:09:52 INFO: Synergy 1.4.16 Client on Linux 4.1.12-rt13-CX9020-9+ #1 PREEMPT RT Fri Apr 29 01:09:47 CEST 2016 armv7l
	/build/synergy-M6oALT/synergy-1.4.16/src/lib/synergy/CClientApp.cpp,157
2016-05-11T23:09:52 DEBUG1: starting client
	/build/synergy-M6oALT/synergy-1.4.16/src/lib/synergy/CClientApp.cpp,633
2016-05-11T23:09:52 DEBUG: XOpenDisplay(":0")
	/build/synergy-M6oALT/synergy-1.4.16/src/lib/platform/CXWindowsScreen.cpp,918
2016-05-11T23:09:52 DEBUG1: thread 0x00000002 entry
	/build/synergy-M6oALT/synergy-1.4.16/src/lib/mt/CThread.cpp,155
X Error of failed request:  BadValue (integer parameter out of range for operation)
  Major opcode of failed request:  53 (X_CreatePixmap)
  Value in failed request:  0x0
  Serial number of failed request:  16
  Current serial number in output stream:  17
```

**Comment by XinyuHou** [comment]
This looks like a support ticket. Please contact our help desk.
https://synergy-project.org/contact/

**Comment by pabloinigoblasco** [comment]
Hello there, It looks like this problem is still happening even using the latest version of the code from github.
I am working with ubuntu 16.04 in an embedded arm machine (odroid xu4)

geus@odroidxu4:~/synergy/bin$ synergyc --version
synergyc 1.6.2, protocol version 1.5

Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.1 LTS
Release:	16.04
Codename:	xenial


geus@odroidxu4:~/synergy/bin$ synergyc -f --name odroidxu4 192.168.2.60
2016-09-18T23:16:34 DEBUG: XOpenDisplay(":0.0")
	/build/synergy-LqIUOl/synergy-1.6.2/src/lib/platform/XWindowsScreen.cpp,920
X Error of failed request:  BadValue (integer parameter out of range for operation)
  Major opcode of failed request:  53 (X_CreatePixmap)
  Value in failed request:  0x0
  Serial number of failed request:  16
  Current serial number in output stream:  17

**Comment by nlyan** [comment]
@pabloinigoblasco the command line output you pasted shows that you are using Synergy version 1.6.2. This particular bug was fixed in v1.8.2
